test/FractionalGovernor.sol

```diff
@@ -1,5 +1,5 @@
 // SPDX-License-Identifier: UNLICENSED
-pragma solidity ^0.8.13;
+pragma solidity >=0.8.10;
```

Comment:
Can I get a quick TL;DR why this changed?

Reply:
Sorry, should have made a note about this. Basically, we had to do it because otherwise we couldn't build the project. We'd get `Discovered incompatible solidity versions` errors. Portions of aave's v3 code -- specifically: portions that we have to import and compile -- are hard-coded to `0.8.10`, e.g. [1](https://github.com/aave/aave-v3-core/blob/f3e037b3638e3b7c98f0c09c56c5efde54f7c5d2/contracts/protocol/libraries/types/ConfiguratorInputTypes.sol#L2), [2](https://github.com/aave/aave-v3-core/blob/f3e037b3638e3b7c98f0c09c56c5efde54f7c5d2/contracts/protocol/pool/PoolConfigurator.sol#L2), [3](https://github.com/aave/aave-v3-core/blob/f3e037b3638e3b7c98f0c09c56c5efde54f7c5d2/contracts/protocol/libraries/math/WadRayMath.sol#L2)

Reply:
Got it, thanks for the explanation.